src/BIP32.php

```diff
@@ -161,7 +161,7 @@ public static function CKD($master, $address_definition, $generated = array())
         }
 
         if (!isset($data)) {
-            return false;
+            throw new \Exception(); // I don't think this can happen
```

Comment:
hmm, yea I forgot, I wanted to comment on this one.

I think it can be removed since `$private_key` or `$public_key` should always be non-null or otherwise the `BIP32::import` or `BitcoinLib::private_key_to_public_key` would already throw an exception.